src/PortfolioContainer/Home/Home.tsx

```diff
@@ -20,8 +20,8 @@ export default function Home(props: ScreenProps) {
       <div className="home-container"
       // id={id}
       >
-        <ScreenHeading props={screenHeadingProps} />
         <HeaderTest />

```

Comment:
What is HeaderTest doing? Looks like you have two headers. Should one be renamed to something more fitting?